### DISCUSSION TITLE
FM broadcast radio

### DISCUSSION THREAD

**Discussion by lohtse** [egzumer/uv-k5-firmware-custom#430] [Q&A]
Just combiled a new version of the FW which now has the "FM radio band selection" but how we use and set the ranges we may wish too use???

**Comment by lohtse** [comment]
Oh never mind found it by reading the code..



Once in FM Radio mode  press F+1 until you find the band you want..

is it possibe to add a 64-108 range please?

**Comment by egzumer** [comment]
Yes and no. Those bands are what is offered by the Bk1080 chip. In theory it seems that you can set 108M frequency even if you choose the lowest range, but I won't attempt to play with this. If you want to know more study the BK1080 reference manual. What is important for me is that I can now scan proper range which in Europe is 87.5-108.

**Reply by lohtse** [reply]
Ok thank you.... Actually figured out how to do it and works fine(was already implimented on 1of11s BTW)
